src/apps/lwaftr/binding_table.lua

```diff
@@ -245,8 +201,7 @@ function BindingTable:lookup_psid(ipv4, port)
 end
 
 function BindingTable:get_br_address(i)
-   assert(i<self.br_address_count)
-   return self.br_addresses[i].addr
+   return self.br_addresses[i+1]
```

Comment:
br_addresses is no longer a raw zero-indexed pointer but an "ffi array", which is 1-indexed and which checks its bounds, so the check is no longer needed.